project.clj

```diff
@@ -1,11 +1,27 @@
-(defproject tentacles "0.4.0"
+(def project-version (if-let [build-num (System/getenv "CIRCLE_BUILD_NUM")]
+                       (str "0.2." build-num)
+                       "0.2.0-SNAPSHOT"))
```

Comment:
Any reason we're going with `0.2.0`? I see that https://github.com/circleci/circle/blob/master/project.clj#L115 is `0.2.4-xxx` should that patch version be the basis of our new "master"?

Reply:
it's a snapshot version when someone is building locally.
when it's build in CI, it's going to be `0.2.<build-num>`, build-num is already 15, so when we done we'll have a final version something like `0.2.16`, would it work?

Reply:
Oh, sorry I misread the code. But now that I see it, shouldn't the non-snapshot version be `0.2.4-build-num` since it builds off of `0.2.4`? `0.2.15	` would be confusing, I think.

Reply:
hm, my though was that we continue growing `0.2.x` line, so as long as next version is bigger then `0.2.4` we should be fine, no?